addons/mail/tests/test_mail_template.py

```diff
@@ -22,6 +22,24 @@ def setUpClass(cls):
             'auto_delete': True,
             'model_id': cls.env.ref('base.model_res_partner').id,
         })
+        cls.mail_template_mso = cls.env['mail.template'].create({
```

Comment:
I think we could create it directly in the test about comment preserve. No need to "bloat" the generic data creation with something used in a single test. Having a smaller set allows to have the minimum amount of data to re-use.

Reply:
:+1: